README.md

```diff
@@ -155,7 +155,7 @@ OPTIONS:
         --verbose          Show verbose output
         --working-dir      The path to the directory containing the git repository. Defaults to the current directory
     -n, --name             The name of the stack. Must be unique
-    -s, --source-branch    The source branch to use for the new branch. Defaults to the default branch for the repository
+        --source-branch    The source branch to use for the new branch. Defaults to the default branch for the repository
```

Comment:
Removed the short option here as it clashes with the more common use for selecting a stack. In this case there isn't an option to select a stack, but don't want things to be confusing.